accelerator/roles/intel/vars/main.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+ansible_become: true
+ansible_become_user: root
+
+gaudi_device_pattern: "Processing accelerators: Habana Labs Ltd."
+
+synapseai_apt_key_url: "https://vault.habana.ai/artifactory/api/gpg/key/public"
+
+synapseai_apt_repo_source: "deb https://vault.habana.ai/artifactory/debian jammy main"
+
+habana_packages:
+  - habanalabs-container-runtime
+  - habanalabs-dkms
+  - habanalabs-firmware
+  - habanalabs-firmware-tools
```

Comment:
accelerator is not meant to install formware packages. Omnia does not support firmware installation hence please remove this from the list

Reply:
This one also cannot be removed in addition to fw utility it contains hl-smi which we rely on to monitor the device and read metrics